main.tf

```diff
@@ -2,18 +2,23 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.0"
+      version = "~> 5.0"
     }
   }
 }
 
 # Subspace will prompt for credentials if they are not found in ~/.aws/credentials
 provider aws {
   region = var.aws_region
-  profile = "subspace-${var.project_name}"
```

Comment:
is this no longer the preferred way?  Subspace has some code to automatically create this profile

Reply:
1. it didn't work for me no matter what I named the profile in my .aws/credentials. 
2. We're not doing it this way on oxenwagen anymore, and when we use TF cloud the aws creds get set as vars on the tf cloud workspace and are pulled from there.

Oxenwagen provider config:
```
provider aws {
  region = var.aws_region
  default_tags {
    tags = {
      Environment = var.project_environment
      Project     = var.project_name
    }
  }
}
```